website/docs/ref/vite-plugin.md

```diff
@@ -0,0 +1,45 @@
+# Vite Plugin
```

Comment:
Just realized that the new page is not displaying in the left menu. Please add a reference in the `website/sidebars.ts` file

Reply:
done